parsl/executors/high_throughput/executor.py

```diff
@@ -569,11 +570,10 @@ def submit(self, func, resource_specification, *args, **kwargs):
         Returns:
               Future
         """
-        if resource_specification:
-            logger.error("Ignoring the resource specification. "
-                         "Parsl resource specification is not supported in HighThroughput Executor. "
-                         "Please check WorkQueueExecutor if resource specification is needed.")
-            raise UnsupportedFeatureError('resource specification', 'HighThroughput Executor', 'WorkQueue Executor')
+        if 'app_type' in resource_specification and len(resource_specification) > 1:
```

Comment:
this can go back to the original test now, right?

Reply:
Change reverted.